firstque.py

```diff
@@ -0,0 +1,29 @@
+password = input()
+
+upper_case = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']
```

Comment:
Here u need not to define these variables
actually python has a inbuilt method to check 
you can use " i.isupper()" to check